docker/nginx/redirects-301.map

```diff
@@ -1,22 +1,192 @@
+/services/hdfs/v2.0.0-2.6.0-cdh5.11.0/img/hdfs-service-gui2.gif /services/hdfs/v2.0.0-2.6.0-cdh5.11.0/img/hdfs-service-gui2.png;
+/services/hdfs/v2.0.0-2.6.0-cdh5.11.0/img/hdfs-service-gui3.gif /services/hdfs/v2.0.0-2.6.0-cdh5.11.0/img/hdfs-service-gui3.png;
+/services/hdfs/vhdfs_1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui.gif /services/hdfs/v1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui.png;
+/services/hdfs/vhdfs_1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui2.gif /services/hdfs/v1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui2.png;
+/services/hdfs/vhdfs_1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui3.gif /services/hdfs/v1.3.0-2.6.0-cdh5.9.1/img/hdfs-service-gui3.png;
+/services/kafka/2.0.1-0.11.0/api-reference/quick-start.md/ /services/kafka/2.0.1-0.11.0/quick-start/;
+/services/kafka/2.0.1-0.11.0/install/api-reference.md/ /services/kafka/2.0.1-0.11.0/api-reference/;
+/services/kafka/2.0.1-0.11.0/managing/configure.md/ /services/kafka/2.0.1-0.11.0/managing/;
+/services/kafka/2.0.2-0.11.0/api-reference/quick-start.md/ /services/kafka/2.0.2-0.11.0/quick-start/;
+/services/kafka/2.0.2-0.11.0/install/api-reference.md/ /services/kafka/2.0.2-0.11.0/api-reference/;
+/services/kafka/2.0.2-0.11.0/managing/configure.md/ /services/kafka/2.0.2-0.11.0/managing/;
+/services/kafka/2.0.3-0.11.0/api-reference/quick-start.md/ /services/kafka/2.0.3-0.11.0/quick-start/;
+/services/kafka/2.0.3-0.11.0/install/api-reference.md/ /services/kafka/2.0.3-0.11.0/api-reference/;
+/services/kafka/2.0.3-0.11.0/managing/configure.md/ /services/kafka/2.0.3-0.11.0/managing/;
+/services/kafka/2.0.4-1.0.0/managing/configure.md/ /services/kafka/2.0.4-1.0.0/managing/;
+/services/kafka/2.0.4-1.0.0/troubleshooting/managing.md/ /services/kafka/2.0.4-1.0.0/managing/#extend-the-kill-grace-period/;
+/services/kafka/configure/ /services/kafka/kafka-auth/;
+/services/kafka/connecting-clients/ /services/kafka/2.0.4-1.0.0/connecting-clients/;
+/services/kafka/v2.0.0-0.11.0/api-reference/quick-start.md/ /services/kafka/v2.0.0-0.11.0/api-reference/;
+/services/kafka/v2.0.0-0.11.0/install/api-reference.md/ /services/kafka/v2.0.0-0.11.0/api-reference/;
+/services/kafka/v2.0.0-0.11.0/managing/configure.md/ /services/kafka/v2.0.0-0.11.0/managing/;
+/services/marathon/ /services/marathon-lb/;
+/services/overview/ /services/;
+/services/spark/v1.1.0-2.1.1/img/spark-gui-install.gif /services/spark/v1.1.0-2.1.1/img/spark-gui-install.png;
+/services/spark/v1.1.1-2.2.0/img/spark-gui-install.gif /services/spark/v1.1.1-2.2.0/img/spark-gui-install.png;
+/services/spark/v2.0.1-2.2.0-1/img/spark-gui-install.gif /services/spark/v2.0.1-2.2.0-1/img/spark-gui-install.png;
+/tutorials/ /latest/tutorials/;
+/tutorials/launch-docker-container-on-mesosphere/ /latest/deploying-services/creating-services/deploy-docker-app/;
+/usage/cli/ /latest/cli/;
+/usage/managing-services/config/ /latest/deploying-services/config-universe-service/;
+/s/downloads/ /s/;
```

Comment:
For both line 159 & 160, `https://docs.mesosphere.com/s/` does not exist, so the redirect won't work. I think you were trying to redirect (`https://support.mesosphere.com/s/downloads/` to `https://support.mesosphere.com/s/`). Since this link is not under `docs.mesosphere`, you won't be able to redirect this way.

Reply:
Removed these entries.